app/routes/models/task-list.js

```diff
@@ -42,24 +42,27 @@ const updateSections = (agreement) => {
   const fundingSection = sections.find(x => x.name === 'Choose your funding')
   const fundingTask = fundingSection.tasks.find(x => x.name === 'Choose funding option')
 
-  // land section complete but funding not set yet
-  if (land.isLandCorrect != null && land.hasManagementControl != null && !funding.length) {
+  // land section complete
+  if (land.landComplete) {
```

Comment:
Not required as above if it's not complete you return from the function

Reply:
Agreed, good spot.  Will be addressed in next PR when this function is completely re-written.